Android/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required (VERSION 2.8.7)
+project(Cuberite)
+include(ExternalProject)
+
+externalproject_add(lua_native
+	SOURCE_DIR ../lib/lua
+	CMAKE_GENERATOR "Visual Studio 14 2015"
+	CMAKE_ARGS -DANDROID_CROSSCOMPILE=TRUE
+	BUILD_COMMAND
+		${CMAKE_COMMAND} --build <BINARY_DIR> --config Release
+	INSTALL_COMMAND ""
+)
+externalproject_add(tolua_native
+	SOURCE_DIR ../lib/tolua++
```

Comment:
I’m not sure.
•	If you mean how it’s doing the referencing, I’m still not sure.
•	If you mean it’s not referencing anything but that I’m saying that it is, then I’m confused.